src/AltinnCore/Designer/TypedHttpClients/AltinnStorage/AltinnApplicationStorageService.cs

```diff
@@ -0,0 +1,43 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Models;
+
+namespace AltinnCore.Designer.TypedHttpClients.AltinnStorage
+{
+    /// <summary>
+    /// AltinnApplicationStorageService
+    /// </summary>
+    public class AltinnApplicationStorageService : IAltinnApplicationStorageService
```

Comment:
Could we reconsider the name? Seems a bit long. Altinn is kinda implicit given the namespace and the rest of the solution so that could maybe be removed?

Reply:
Have you taken into consideration that there is another ApplicationMetadataService implemented in Altinn.Common ? IApplication/ApplicationAppSI/ApplicationStudioSI.  Is it an active choice that we have split appMetadata operations in two seperate services?

Reply:
The reason AltinnApplicationStorageService and ApplicationMetaDataService exists is because of a function in the legacy DeployController (which is now deleted).

[Function in DeployController which has been extracted into several classes](https://github.com/Altinn/altinn-studio/blob/589519850f15441e7988f817cebd8c2eafe53b30/src/AltinnCore/Designer/Controllers/DeployController.cs#L234)

The ApplicationMetadataService implements the same logic as the DeployController function mentioned above. It also uses the same repository as ApplicationStudioSI so there is an argument there that one could implement the repository functions that ApplicationMetadataService uses in ApplicationStudioSI and ApplicationMetadataService injects the ApplicationStudioSI instead.

I am open to moving and refactoring the files inside Designer to better fit the solution structure.

However, I did not want to create a mess with the existing files and decided to isolate all the files inside the Designer project. I am also unsure if we want one Service to inject another Service, instead of injecting a Repository.